cmd/process-exporter/main.go

```diff
@@ -293,6 +293,8 @@ func main() {
 			"if a proc is tracked, track with it any children that aren't part of their own group")
 		threads = flag.Bool("threads", true,
 			"report on per-threadname metrics as well")
+		smaps = flag.Bool("gather-smaps", false,
```

Comment:
Can you change the default to true please?  I'd rather this feature be opt-out, since it seems valuable and only costs significant performance on older systems.  We can flag this in the changelog.